app/views/fs/messages/_submission.html.erb

```diff
@@ -0,0 +1,57 @@
+<style>
+  th {
+    font-weight: normal;
+    text-align: left;
+  }
+  th, td {
+    padding: 0.25rem 2rem 0.25rem 0;
+  }
+</style>
+
+<table>
+  <tbody>
+    <tr>
+      <th>Identifikátor subjektu:</th>
+      <td>
+        <b><%= message.metadata['dic'] %></b>
+      </td>
+    </tr>
+    <tr>
+      <th>Daňový subjekt:</th>
+      <td>
+        <b><%= message.recipient_name %></b>
+      </td>
+    </tr>
+    <tr>
+      <th>Evidenčné číslo podania:</th>
+      <td>
+        <b><%= message.metadata['fs_message_id'] %></b>
+      </td>
+    </tr>
+    <tr>
+      <th>Dátum podania:</th>
+      <td>
+        <b>
+          <time class="md:hidden" datetime="<%= message.delivered_at %>">
+            <%= ApplicationController.helpers.nice_datetime_with_time(message.delivered_at, keep_year: true) %>
```

Comment:
Toto sa neda volat priamo cez `nice_datetime_with_time` ?

Reply:
Nie, lebo to nevolame z controllera, tak nie su bezne dostupne veci vo view k dispozicii